src/network/index.js

```diff
@@ -1008,6 +1027,15 @@ realityEditor.network.onAction = function (action) {
                 realityEditor.network.reloadFrame(thisAction.reloadFrame.object, thisAction.reloadFrame.frame, thisAction);
             }, 500);
         }
+        
+        // todo Steve: repositioned a frame
+        let params = {
+            objectKey: thisFrame.objectId,
+            frameKey: thisFrame.uuid,
+            frameType: thisFrame.src,
+        };
+        let avatarId = realityEditor.avatar.getAvatarObjectKeyFromSessionId(thisAction.lastEditor);
+        realityEditor.ai.onFrameRepositioned(params, avatarId);
```

Comment:
Similar to above comment, it might be better to subscribe to an event (though looks like one doesn't exist for this yet). However, I'd also recommend you move this code block into the 

```
if (thisKey === 'ar' &&
    propertiesToIgnore.indexOf('ar.x') > -1 &&
    propertiesToIgnore.indexOf('ar.y') > -1) {
    ...
```
part of realityEditor.network.reloadFrame, as where you have it now might get triggered by various updates within the frame, not just repositioning.

Reply:
The internal reposition mechanism changed and I can't get the original & updated frame positions from here. Will temporarily disable the "frameRepositioned" event.